packages/common/src/api/tan-query/wallets/useTokenBalance.ts

```diff
@@ -50,21 +51,25 @@ export const getTokenBalanceQueryKey = (
  */
 export const useTokenBalance = ({
   mint,
+  userId: userIdParam,
   isPolling,
   pollingInterval = 1000,
   ...queryOptions
 }: {
   mint: string
+  userId?: ID
   isPolling?: boolean
   pollingInterval?: number
 } & QueryOptions) => {
   const { audiusSdk, env } = useQueryContext()
-  const { data: user } = useCurrentAccountUser()
-  const ethAddress = user?.wallet ?? null
+  const { data: user } = useUser(userIdParam, { enabled: !!userIdParam })
```

Comment:
feel like there may be some tanquery `select` stuff we can do here to just get the `userId`